images/ose-cli-artifacts.yml

```diff
@@ -9,23 +9,22 @@ content:
     ci_alignment:
       streams_prs:
         ci_build_root:
-          stream: rhel-8-golang-1.20-ci-build-root
+          stream: rhel-9-golang-ci-build-root
 distgit:
-  branch: rhaos-{MAJOR}.{MINOR}-rhel-8
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
   component: ose-cli-artifacts-container
   namespace: containers
 enabled_repos:
-- rhel-8-appstream-rpms
-- rhel-8-baseos-rpms
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
 for_payload: true
 from:
   builder:
-  - stream: golang-1.20
```

Comment:
bot always tries to downgrade our go version automatically https://github.com/openshift/oc/pull/1703, is this stream the reason of this?.

Reply:
ci_build_root:
          stream: rhel-8-golang-1.20-ci-build-root
is the reason